### DISCUSSION TITLE
About the new layout

### DISCUSSION THREAD

**Discussion by ojvribeiro** [elk-zone/elk#861] [General]
I'd say I prefer more the previows Twitter-like layout with the search bar on the right and a highlighted Compose button. Now we have everything put toghether on the left side and a huge empty space on the right. 😭 

![image](https://user-images.githubusercontent.com/8026741/211169574-c198221a-526c-4409-baed-0a91b7d3461c.png)

A sugestion of mine would be keep the left side to things related to the user, like Notifications, Favorites, etc., and the Explore related items on the right, along with the search bar. And yeah, that's what Twitter does. What can I say? It just works 🤷‍♂️ 😄

**Comment by patak-cat** [comment]
Hey @ojvribeiro! Thanks for the honest feedback. We aren't married with some of the decisions we made for 0.2.0, and we are going to continue reviewing the design. We want to take the opportunity to try a few ideas before we have more users and gets more difficult to play with them.

About the search, the idea is that it makes the sidebar stable when you resize it. The right sidebar will have content later on also. And the scrollbar you're seeing now shouldn't be there soon. We're also going to try adding back a bright Compose button. I think the best is to wait a bit more until the new design is stable before comparing it to the previous one.

**Reply by ojvribeiro** [reply]
You're absolutely right, there's a lot to experiment and see what works for everyone. Anyway I'm loving Elk and I'm looking forward to use it as my primary social network.

**Comment by bigshans** [comment]
Right! The right column leaves too much useless space and makes the center column small. I think the center column is best set to `720px`.

**Reply by pdiamond-op** [reply]
On the desktop I would also prefer a wider layout.

**Comment by pdiamond-op** [comment]
I would love to see the posts in full width on mobile. There is so much wasted space on the left side causing extra scrolling on mobile devices where there is already less screen real estate. Megalodon uses full width posts and it allows the images to be larger and generally allows more posts on screen (except that they put the name of the app at the top, which takes up too much space). 

![Screenshot_20230117_114125_Megalodon](https://user-images.githubusercontent.com/113446450/212963094-841a41d4-80a8-4ab2-85cf-0bb163a52b86.jpg)

![Screenshot_20230117_114058_Brave](https://user-images.githubusercontent.com/113446450/212963121-2c1aa12b-13ea-41e1-98c7-7a96057ea879.jpg)
